pkg/dataplane/linux/dataplane_linux.go

```diff
@@ -76,8 +78,10 @@ func (d *linuxDataplane) DoNetworking(
 	err = ns.WithNetNSPath(args.Netns, func(hostNS ns.NetNS) error {
 		veth := &netlink.Veth{
 			LinkAttrs: netlink.LinkAttrs{
-				Name: contVethName,
-				MTU:  d.mtu,
+				Name:        contVethName,
+				MTU:         d.mtu,
+				NumTxQueues: d.queues,
+				NumRxQueues: d.queues,
```

Comment:
What was the previous default for these fields, when we weren't setting them explicitly?

I guess it was 1, and so by default this PR doesn't change that, but adds the capability to do that by configuring `n_queues`.  Is that right?

Reply:
yes